src/cf-layout.less

```diff
@@ -1606,6 +1606,130 @@
 }
 
 
+
+/* topdoc
+  name: Hero
+  family: cf-layout
+  patterns:
+    - name: .hero
+      markup: |
+        <section class="content_hero hero">
+            <div class="hero_wrapper">
```

Comment:
This div also needs the plain `wrapper` class to constrain width on large screens.